src/main/java/nl/tudelft/ti2206/group9/gui/WarningPopup.java

```diff
@@ -0,0 +1,99 @@
+package nl.tudelft.ti2206.group9.gui;
+
+import javafx.event.EventHandler;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Button;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+import javafx.scene.text.Text;
+import javafx.stage.Popup;
+
+import java.util.Arrays;
+
+/**
+ * @author Mathias
+ */
```

Comment:
Can you add a @SuppressWarnings("restriction") here please? :)

<review,organization